consensus/reactor.go

```diff
@@ -545,6 +571,17 @@ OUTER_LOOP:
 		if !peer.IsRunning() || !conR.IsRunning() {
 			return
 		}
+
+		// sleep random amount to give reactor a chance to receive HasProposalBlockPart messages
```

Comment:
these sleeps are at the top of the loop, so effect everything, including catchup logic, which is probably not optimal, but maybe ok.

Reply:
Scale tests show overall improvement, so let's keep it this way for the time being.

However, I've changed the default value in config to 0, as it's still experimental. Folks can try raising it and measuring its impact in their own networks.